src/main/java/org/jabref/model/entry/Month.java

```diff
@@ -97,6 +105,29 @@
         }
     }
 
+    /**
+     * Parses a month having the string in German standard form such as
+     * "Oktober" or in German short form such as "Okt"
+     * 
+     * @param value,
+     *            a String that represents a month in German form
+     * @return the corresponding month instance, empty if input is not in German
+     *         form
+     */
+	private static Optional<Month> parseGermanShortMonth(String value) {
+        if ("Mae".equals(value)) {
```

Comment:
What if it is "mae" in lowercase?

Reply:
The implementation should use `equalsIgnoreCase`.

Reply:
Please fix indent.